BurstBotShared/Services/SkiaService.cs

```diff
@@ -14,6 +15,12 @@ public static class SkiaService
     private const int Quality = 95;
     private const int Padding = 50;
```

Comment:
Hmm, I would personally rename SkiaService to something more future proof like AssetRendererService, since maybe in the future you would something other than Skia, but the function still remains the same that is to render assets

Reply:
Don't you think AssetRendererService is too generic in that case? For example, what kind of assets? Graphic files or text files?